Vagrantfile

```diff
@@ -1,7 +1,11 @@
-
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/xenial64"
+  config.vm.provision :shell, path: "setup.sh", :privileged => false 
+  #config.vm.network :public_network, ip: "192.168.50.6"
+  config.vm.network :private_network, ip: "192.168.50.6"
+  config.vm.network :forwarded_port, guest: 5000, host: 5000
```

Comment:
Is there a reason you  you choose this IP address?

Reply:
No. Arbitrary IP in the private network address scope to enable comm between host and guest,
Found it unnecessary as the forwarded_port config is not dependent on a private network IP address.
Removed this line in latest commit.